src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -163,9 +191,9 @@ def is_used(self) -> bool:
     def is_alive(self) -> bool:
         # Unfortunately, with and without TZ information is required for compare
         # or exceptions are generated
-        ten_minutes_ago_no_tz = datetime.datetime.utcnow() - datetime.timedelta(
-            minutes=10
-        )
+        ten_minutes_ago_no_tz = datetime.datetime.now(
```

Comment:
This forces a timezone into `ten_minutes_ago_no_tz` verses `ten_minutes_ago`.  If you need a timezoned version, use `ten_mintues_ago`.

Reply:
So I just added a comment in another spot of the code, but I only just realized that self.timestamp is updated every ten minutes. I was treating it as a 'created_timestamp.' 

I believe I need to add a new field that's set at deployment time for 'created_timestamp' to check in the is_used() function. Is this line of thinking correct?